### ISSUE TITLE
[Bug] Wallet Not Syncing

### ISSUE THREAD

**Issue by Tiwka3000** [Chia-Network/chia-blockchain#14924]
### What happened?

hello, the wallet is not syncing in the log folder writes this

### Version

1.7.1

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
:2023-03-28T17:33:36.314 wallet chia.wallet.wallet_node    : WARNING  querying DNS introducer failed: The resolution lifetime expired after 30.343 seconds: Server 192.168.1.1 UDP port 53 answered The DNS operation timed out after 1.999 seconds; Server 192.168.1.1 UDP port 53 answered ; Server 192.168.1.1 TCP port 53 answered The DNS operation timed out after 2.000 seconds; Server 192.168.1.1 UDP port 53 answered ; Server 192.168.1.1 TCP port 53 answered The DNS operation timed out after 2.000 seconds; Server 192.168.1.1 UDP port 53 answered ; Server 192.168.1.1 TCP port 53 answered The DNS operation timed out after 2.000 seconds; Server 192.168.1.1 UDP port 53 answered ; Server 192.168.1.1 TCP port 53 answered The DNS operation timed out after 2.000 seconds; Server 192.168.1.1 UDP port 53 answered ; Server 192.168.1.1 TCP port 53 answered The DNS operation timed out after 2.000 seconds; Server 192.168.1.1 UDP port 53 answered ; Server 192.168.1.1 TCP port 53 answered The DNS operation timed out after 2.000 sec
```

**Comment by MumfMeisterT** [comment]
We're working on this issue, for now I would set your DNS to 8.8.8.8

**Comment by MumfMeisterT** [comment]
@xdustinface Shouldn't we be falling back to introducer.chia.net if the seeder fails to resolve?

**Comment by jfikar** [comment]
The wallet is not syncing on Linux cmdline either. I think, the problem are the recent changes in the `config.yaml` file. You can backup the old one and create a fresh one by chia. Or take `chia/util/initial-config.yaml`. I suspect it is these lines:

old not working:
```
full_node:
...
  introducer_peer:
    host: introducer-eu.chia.net
...
```

new working:
```
full_node:
...
  introducer_peer: 
    host: introducer.chia.net
...
```

**Comment by nikaady** [comment]
hey, I deposite xch into my wallet and I can't see any transactions and my wallet balance is 0
also, in the farming part my status is always syncing and not synced again and again. what should I do??
please help me its so emergency. thank you a lot

**Comment by LoudSyncro** [comment]
> The wallet is not syncing on Linux cmdline either. I think, the problem are the recent changes in the `config.yaml` file. You can backup the old one and create a fresh one by chia. Or take `chia/util/initial-config.yaml`. I suspect it is these lines:
> 
> old not working:
> 
> ```
> full_node:
> ...
>   introducer_peer:
>     host: introducer-eu.chia.net
> ...
> ```
> 
> new working:
> 
> ```
> full_node:
> ...
>   introducer_peer: 
>     host: introducer.chia.net
> ...
> ```

Thank you this helped for me. Just discovered full node node syncing automatically any more (not finding any nodes).
Hope in future devs will take care of new / other configs an do migration scripts during update of the software.

**Comment by androiddevkotlin** [comment]
Are you tested this app on windows 11? So many people with the same bugs. Where did you find your developers? In the trash?

**Comment by emlowe** [comment]
Issues with unresolvable introducers blocking syncing has been fixed in 1.8.0. Closing as fixed.

and yes, all our developers were found in the trash.

**Comment by okovtun** [comment]
> We're working on this issue, for now I would set your DNS to 8.8.8.8

Thank you so much, it hepled immidiatelyю
